Bottom: e418488c41c8b5afc2f33c9c6a2dfc8a3b190a3f
Top:    c167165d21a439910b715620b3642f6f149d7666
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 12:41:57 +0000

Add reversed orientation field to face3d structure


---

diff --git a/src/hid/common/face3d.h b/src/hid/common/face3d.h
index 4838c43..c59680d 100644
--- a/src/hid/common/face3d.h
+++ b/src/hid/common/face3d.h
@@ -1,5 +1,6 @@
 typedef struct {
   double nx, ny, nz; /* Face normal?*/
+  bool plane_orientation_reversed;
   GList *contours;
 
   /* For cylindrical surfaces */
